lib/charms/loki_k8s/v1/loki_push_api.py

```diff
@@ -349,6 +352,72 @@ def _promtail_error(self, event):
     )
 ```
 
+## LogForwarder Library Usage
+
+Let's say that we have a workload charm that produces logs to the standard output (stdout),
+and we need to send those logs to a workload implementing the `loki_push_api` interface,
+such as `Loki` or `Grafana Agent`.
+
+Use the `LogForwarder` class by instantiating it in the `__init__` method of the charmed
+operator. There are two ways to provide Loki endpoint(s) to the forwarder. You can let the
```

Comment:
I'd say:
"The easiest way to pass a Loki endpoint to the forwarder is to provide it with the name of an endpoint which uses the loki_push_api interface. The forwarder will automatically extract from there all the data it needs.
Otherwise, you can manually pass a Loki endpoint to which the logs should be forwarded."